resources/lib/modules/globals.py

```diff
@@ -680,12 +680,22 @@ def get_kodi_video_db_connection(self):
             return MySqlConnection(config)
 
     def get_kodi_database_version(self):
-        if self.KODI_VERSION == 17:
+        dbids = []
+        for root, dirs, files in os.walk(tools.translate_path(os.path.join("special://profile/", "Database/"))):
+            for name in files:
+                dbver = re.search(r"MyVideos(\d+)\.db", name)
+                if dbver:
+                    dbids.append(int(dbver.groups()[0]))
+        if dbids:
+            return sorted(dbids)[-1]
```

Comment:
This approach isn't really accurate, as it will return the highest version of the DB, regardless of if it is one we support, negating the need for the checks below.

We have added the check for Nexus (DB version 121) in our development builds, so that will already be implemented in the next release.